### ISSUE TITLE
format_governance

### ISSUE THREAD

**Issue by vturpin** [OceanGlidersCommunity/OG-format-user-manual#55]
Hi,
I raise this issue to discuss the management of the format, controlled vocabularies. 

To make this document lively we need to find procedure to safely but quickly allow evolution of the core document and addition of new element in the vocabs.

Any suggestion from the experienced people ?

**Comment by justinbuck** [comment]
@vturpin  Starting discussion with a proposal.

 We need a current version, and we can then branch it for new iterations, we would formally review and accept the iterations via the OGDMTT (maybe semi-annually initially then annually afterwards). Once a new version is approved this becomes the live version with older versions remaining valid for a time period (3-5 years? ) to allow DACs to catchup .

**Comment by glidermann** [comment]
Agreed. Once this version is approved, we need to explain how the community can make their iterations on GitHub, and how it will be reviewed. Something to propose at this summer's DM meeting?
